### PR TITLE
feature: use deb package instead tar.gz for fabricmanager

### PR DESCRIPTION
**Pull request by janosbinder**
**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:
Fixes #

**Special notes for your reviewer**:

**Release note**:

```feature user

```

### CODE REVIEW

File: resources/download_fabricmanager.sh [hunk 1]
```diff
@@ -23,8 +23,8 @@ mkdir -p /tmp/nvidia
...
 # Download Fabric Manager tarball
-OUTDIR=/out/nvidia-fabricmanager/$DRIVER_VERSION
-FABRICMANAGER_ARCHIVE="fabricmanager-linux-$ARCH_TYPE-$DRIVER_VERSION-archive"
-FABRICMANAGER_URL="https://developer.download.nvidia.com/compute/cuda/redist/fabricmanager/linux-$ARCH_TYPE/${FABRICMANAGER_ARCHIVE}.tar.xz"
-mkdir -p "$OUTDIR"
-wget --directory-prefix="${OUTDIR}" "${FABRICMANAGER_URL}"
\ No newline at end of file
+wget -O /tmp/keyring.deb https://developer.download.nvidia.com/compute/cuda/repos/debian12/x86_64/cuda-keyring_1.1-1_all.deb && dpkg -i /tmp/keyring.deb
+apt-get update
+#apt-get install -V nvidia-open-"$DRIVER_BRANCH"
```
Comment (dhague) [inline resources/download_fabricmanager.sh:28]: Maybe remove this line instead of commenting out?

File: resources/download_fabricmanager.sh [hunk 2]
```diff
@@ -23,8 +23,8 @@ mkdir -p /tmp/nvidia
 pushd /tmp/nvidia
 
 # Download Fabric Manager tarball
-OUTDIR=/out/nvidia-fabricmanager/$DRIVER_VERSION
-FABRICMANAGER_ARCHIVE="fabricmanager-linux-$ARCH_TYPE-$DRIVER_VERSION-archive"
-FABRICMANAGER_URL="https://developer.download.nvidia.com/compute/cuda/redist/fabricmanager/linux-$ARCH_TYPE/${FABRICMANAGER_ARCHIVE}.tar.xz"
-mkdir -p "$OUTDIR"
-wget --directory-prefix="${OUTDIR}" "${FABRICMANAGER_URL}"
\ No newline at end of file
+wget -O /tmp/keyring.deb https://developer.download.nvidia.com/compute/cuda/repos/debian12/x86_64/cuda-keyring_1.1-1_all.deb && dpkg -i /tmp/keyring.deb
```
Comment (dhague) [inline resources/download_fabricmanager.sh:26]: We need to be a bit careful in the future when Garden Linux is based on a Debian version >12. Fine for now though.

File: resources/download_fabricmanager.sh [hunk 3]
```diff
@@ -23,8 +23,8 @@ mkdir -p /tmp/nvidia
...
+apt-get update
+#apt-get install -V nvidia-open-"$DRIVER_BRANCH"
+apt-get install -V nvidia-fabricmanager-"$DRIVER_BRANCH" nvidia-fabricmanager-dev-"$DRIVER_BRANCH"
```
Comment (dhague) [inline resources/download_fabricmanager.sh:29]: Is `nvidia-fabricmanager-dev` needed? I would have expected just `nvidia-fabricmanager`

File: resources/download_fabricmanager.sh (hunk 2 above)
Comment (janosbinder) [reply resources/download_fabricmanager.sh:26]: Actually debian12 is the latest one available there. There is debian11, debian10 and ubuntu ones.

File: resources/download_fabricmanager.sh (hunk 3 above)
Comment (janosbinder) [reply resources/download_fabricmanager.sh:29]: Will be removed.

File: resources/download_fabricmanager.sh (hunk 1 above)
Comment (janosbinder) [reply resources/download_fabricmanager.sh:28]: Will be removed.

**Pull request merged**